src/python/WMCore/Services/DBS/DBSErrors.py

```diff
@@ -58,3 +60,53 @@ class DBSReaderError(DataMgmtError):
     """
     def __init__(self, msg, **data):
         DataMgmtError.__init__(self, msg, 1002, **data)
+
+def getServerCode(httpError):
+    """
+    Return appropriate message based on giver HTTPError server code
+    """
+    dbsCodes = "https://github.com/dmwm/dbs2go/blob/master/dbs/errors.go"
+    if hasattr(httpError, "server_code"):
+        return "DBS server code {}. Please refer to {} for details".format(httpError.server_code, dbsCodes)
+    elif isinstance(httpError, dict) and 'code' in httpError:
+        return "DBS server code {}. Please refer to {} for details".format(httpError['code'], dbsCodes)
+    return "DBS HTTPError '{}' does not provide DBS server code".format(httpError)
+
+
+class DBSError():
+    """
+    DBSError provides generic interface for DBS (Go-based) errors
+    """
+    def __init__(self, ex):
+        try:
+            if hasattr(ex, "body"):
+                # case of DBSClient HTTPError
+                self.data = json.loads(ex.body)[0]
+            else:
+                self.data = json.loads(ex)[0]
+        except:
+            self.data = str(ex)
+
+    def Code(self):
```

Comment:
As already mentioned in my previous review, method names should start with lower case, see:
https://github.com/dmwm/WMCore/blob/master/standards/.pylintrc#L97-L98

and I would suggest to explicitly call it a getter method, e.g.: `def getCode`. Same applies to the methods below.

Reply:
done